control_panel_api/aws.py

```diff
@@ -46,6 +46,20 @@ def put_bucket_logging(self, name, target_bucket, target_prefix):
                 }
             })
 
+    def put_bucket_tagging(self, name, tags):
+        """Put bucket tagging
+
+        :param name: Bucket name
+        :param tags: Tags {key: value} e.g. {'buckettype': 'datawarehouse'}
+        :type name: str
+        :type tags: dict
+        """
+        self._do('s3', 'put_bucket_tagging',
```

Comment:
Bear in mind that merge this would break if you don't update the CP-API IAM role so that it has the permission to perform this action.

You need something like [this](https://github.com/ministryofjustice/analytics-platform-ops/pull/123) with the right S3 action and to do a `terraform apply` in dev/alpha.